### ISSUE TITLE
[4.0] linting SCSS

### ISSUE THREAD

**Issue by C-Lodder** [joomla/joomla-cms#30032]
Is Joomla no longer linting the SCSS?

I only ask as there are currently 136 issues in Atum's SCSS raised by the linter.

If Joomla doesn't want to lint the SCSS, then the following files can be removed from the repo:

- `Gemfile`
- `Gemfile.lock`

**Comment by brianteeman** [comment]
We should be. I spent ages fixing all the errors a few months ago

**Comment by Hackwar** [comment]
will look into this.

**Comment by Hackwar** [comment]
This is a bit strange. As far as I can tell, Hound should be working, at least from a review of our setup. It also threw errors in earlier PRs, specifically those where you, @brianteeman, fixed a bunch of stuff. However in other cases it stayed silent, for example here https://github.com/joomla/joomla-cms/pull/28286

I can only guess that Hound was misconfigured at one point. I'm generally not happy with Hound here, because it only checks files that are touched in the current PR and thus somewhere stuff like this slips through. It also uses outdated software.

Long story short: Yes, Joomla still wants and does lint its SCSS. Yes, there are currently issues which we have to solve. The solution to our issues is to switch to a different linter and to move that into Drone.

**Comment by Hackwar** [comment]
This should be fixed by #30196
